### PR TITLE
:sparkles: updated cities1000, geoSpatial search

### PR DESCRIPTION
**Pull request by nabilashraf**
Hi @zeke , 

Here are the changes: 

1. Changed the build script to output `cities.pbf`, there was an error in parsing and with pbf format it works just fine. 
2. Updated the cities1000 package with the newer version I created and maintaining it. 
3. Added lon/lat structure for GeoSpatial Search in mongodb
4. Added cityId so anyone who wants to use it against openWeatherMap bulk db etc can do so by matching the cityId.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-Derived from the [cities1000](https://www.npmjs.com/package/cities1000) npm package, which in turn came from [geonames.org data](http://download.geonames.org/export/dump/).
+Derived from the [cities-with-1000](https://www.npmjs.com/package/cities-with-1000) npm package, which in turn came from [geonames.org data](http://download.geonames.org/export/dump/).
 
 ## Installation
 
 Download node at [nodejs.org](http://nodejs.org) and install it, if you haven't already.
 
 ```sh
-npm install all-the-cities --save
+npm install all-the-cities-mongodb --save
```
Comment (nabilashraf) [inline README.md]: Please change it to `all-the-cities` 

Forgot to edit my package source.

**Comment by zeke** [comment]
Thanks for the PR! I created and merged https://github.com/zeke/all-the-cities/pull/13 which adds a Travis config for runnings tests on PRs. Can you pull in the latest changes to your branch so we can make sure the tests are passing here?

**Comment by nabilashraf** [comment]
> 
> 
> Thanks for the PR! I created and merged #13 which adds a Travis config for runnings tests on PRs. Can you pull in the latest changes to your branch so we can make sure the tests are passing here?

I pulled in the latest changed to feature branch but the tests didn't pass because of package-lock.json file is not present. Let me add that

**Comment by nabilashraf** [comment]
All checks have passed, updated the test case and package-lock.json to the latest package.json file @zeke

**Comment by zeke** [comment]
Published!

- https://github.com/zeke/all-the-cities/releases/tag/v3.0.0
- https://www.npmjs.com/package/all-the-cities

```
~/git/zeke/all-the-cities master
$ np major

Publish a new version of all-the-cities (current: 2.0.2)

Commits:
- Merge pull request #12 from nabilashraf/update-cities1000  cb6fa56
- updated package-lock and test case  6b8c319
- added package-lock.json  bdc2c25
- Merge pull request #1 from zeke/master  a92a505
- Merge pull request #13 from zeke/add-travis-config  61a8cda
- add travis config  6b0173b
- :memo: Updated README  c5be8be
- :sparkles: updated cities1000, geoSpatial search  d935d10

Commit Range:
v2.0.1...master


  ✔ Prerequisite check
  ✔ Git
  ↓ Cleanup [skipped]
  ✔ Installing dependencies using npm
  ✔ Running tests using npm
  ✔ Bumping version using npm
  ✔ Publishing package using npm
  ✔ Pushing tags
  ✔ Creating release draft on GitHub

 all-the-cities 3.0.0 published 🎉
```

**Pull request merged**